assets/src/stories-editor/blocks/amp-story-page/remove-button-setting.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	PanelBody,
+	Button,
+} from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**.
+ * Add a simple remove page link to sidebar.
+ *
+ * @param {Object} props Component props.
+ * @param {string} props.clientId Client ID.
+ */
+const RemovePageSetting = ( { clientId } ) => {
+	const { removeBlock } = useDispatch( 'core/block-editor' );
+	const pages = useSelect( ( select ) => select( 'core/block-editor' ).getBlockOrder(), [] );
+
+	const removePage = () => {
+		removeBlock( clientId );
+	};
+
+	// Shouldn't allow users to remove the first and only page.
```

Comment:
FWIW, when deleting the first and only page the editor just inserts a new one. So there is always a page.

However, I think it makes sense to not display the link in this case.

Reply:
I know, but it was weird UX that you can keep clicking the delete button and a new one is created.